lua/simple-note.lua

```diff
@@ -97,6 +97,11 @@ M.createNoteFile = function(opts)
     full_path = full_path .. os.date("%A_%B_%d_%Y_%I_%M_%S_%p") .. ".md"
   end
 
+  if vim.fn.filereadable(full_path) == 1 then
+    vim.notify(full_path .. " already exists")
```

Comment:
I would prefer the function to remain silent if the file already exists. 

1) When the function is invoked from the telescope picker, it should be evident that the file already exists just by looking at it. 
2) When the function is invoked from the `SimpleNoteCreate` command, it would be more streamlined to simply open the file if it already exists.